sass/utils/_variables.scss

```diff
@@ -0,0 +1,235 @@
+// H2:    -------  Roboto, Condensed, 22pt       1.375em
+// A:     -------  Roboto, BoldCondensed, 12pt   $font-smaller
+
+// Section 4 Why Choose AppBox
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+// H2:    -------  Raleway, SemiBold, 15pt       $font-regular
+// P:     -------  Roboto, Light, 12pt, 1.67(lh) $font-smaller, $primary-lh
+
+// Section 5 App Reviews
+// H1:    -------  Raleway, Regular, 32pt        $font-larger
+// P:     -------  Roboto, Light, 16pt           $font-large
+// BLOCKQ:-------  Roboto, Italic, 17pt,1.53(lh) $blockquote
+// STRONG:-------  Roboto, Condensed, 20pt       1.25em
+// FIGCAP:-------  Roboto, Regular, 12pt         $font-smaller
+
+// Section 6 Screenshots
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+
+// Section 7 More Apps by AppBox
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+// H2:    -------  Raleway, SemiBold, 15pt       $font-regular
+// P:     -------  Roboto, Condensed, 18pt       1.125em
+// HOVER: -------  Roboto, Light, 12pt, 2(lh)    $font-smaller
+
+// Section 8 Download Our App
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+// STRONG:-------  Raleway, Medium, 32pt
+// P:     -------  Roboto, Light, 16pt           $font-large
+// Google/iOS:---  Roboto, Condensed, 22pt       1.375em
+// Img:   -------  Strasua, Regular, 26pt        1.625em
+
+// Aside
+// LABEL: -------  Raleway, SemiBold, 20pt       1.25em
+// PLACEHOLDER:--  Roboto, Light, 14pt           $font-small
+// BUTTON:-------  Roboto, Medium, 12pt          $font-smaller
+
+// Section 9 Discounted Pricing
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+// P:     -------  Roboto, Light, 14pt,1.375(lh) $font-small, $secondary-lh
+// TH:    -------  Raleway, SemiBold, 15pt       $font-regular
+// TD-SPAN:------  Roboto, BoldCondensed, 26pt   1.625em
+// TD-Price:-----  Roboto, Light, 10pt           0.625em
+// TD:    -------  Roboto, Light, 14pt           $font-small
+// TFOOT: -------  Roboto, Medium, 12pt          $font-smaller
+
+// Section 10 Team Members
+// H1:    -------  Raleway, Regular, 32pt        $font-larger
+// P:     -------  Roboto, Light, 16pt           $font-large
+// H2:    -------  Roboto, Condensed, 20pt       1.25em
+// FIG-P: -------  Roboto, Regular, 12pt         $font-smaller
+
+// Section 11 AppBox Featured In
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+
+// Section 12 App Support
+// H1:    -------  Raleway, Light, 32pt          $font-larger
+// P:     -------  Roboto, Light, 14pt,1.375(lh) $font-small, $secondary-lh
+// H2:    -------  Raleway, SemiBold, 15pt       $font-regular
+// A:     -------  Roboto, Light, 12pt, 1.83(lh) $font-smaller
+// PLACEHOLDER:--  Roboto, Light, 12pt           $font-smaller
+// BUTTON:-------  Roboto, Medium, 12pt          $font-smaller
+
+// Footer
+// LOGO:  -------  Strasua, Regular, 26pt        1.625em
+// COPYRIGHT:----  Roboto, Light, 12pt           font-smaller
```

Comment:
Overall, this is a much more sophisticated config file than I would expect from a team in their second week of using Sass. Kudos!